en/deploy-br-federation-manager.md

```diff
@@ -0,0 +1,235 @@
+    cluster: ${CURRENT_CLUSTER}
+    user: br-federation-member-${DATA_PLANE_SYMBOL}
+    namespace: kube-system
+clusters:
+- name: ${CURRENT_CLUSTER}
+  cluster:
+    certificate-authority-data: ${CLUSTER_CA}
+    server: ${CLUSTER_SERVER}
+users:
+- name: br-federation-member-${DATA_PLANE_SYMBOL}
+  user:
+    token: ${USER_TOKEN_VALUE}
+EOF
+```
+
+- The environment variable `$DATA_PLANE_SYMBOL` represents the name of the data plane cluster. Make sure that you provide a brief and unique name. In the preceding script, you use this variable as the context name for kubeconfig. The context name will be used as `k8sClusterName` in both the `VolumeBackup` and `VolumeRestore` CR.
+
+### Step 1.3: Merge multiple kubeconfig files into one
+
+After following the previous steps to generate kubeconfig, you now have multiple kubeconfig files. You need to merge them into a single kubeconfig file.
+
+Assume that you have 3 kubeconfig files with file paths: `kubeconfig-path1`, `kubeconfig-path2`, `kubeconfig-path3`. To merge these files into one kubeconfig file with file path `data-planes-kubeconfig`, execute the following command:
+
+```shell
+KUBECONFIG=${kubeconfig-path1}:${kubeconfig-path2}:${kubeconfig-path3} kubectl config view --flatten > ${data-planes-kubeconfig}
+```
+
+## Step 2: Deploy BR Federation Manager in the control plane
+
+To deploy the BR Federation Manager, you need to select one Kubernetes cluster as the control plane. The following steps should **only be executed on the control plane**.
+
+### Step 2.1: Create CRD
+
+The BR Federation Manager uses [Custom Resource Definition (CRD)](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/#customresourcedefinitions) to extend Kubernetes. Before using the BR Federation Manager, you must create the CRD in your Kubernetes cluster. This operation only needs to be performed once.
+
+```shell
+kubectl create -f https://raw.githubusercontent.com/pingcap/tidb-operator/master/manifests/federation-crd.yaml
+```
+
+### Step 2.2: Prepare kubeconfig secret
+
+Now that you already have a kubeconfig file of data planes, you need to encode the kubeconfig file into a secret. Take the following steps:
+
+1. Encode the kubeconfig file:
+
+    ```shell
+    base64 -i ${kubeconfig-path}
+    ```
+
+2. Store the output from the previous step in a secret object.
+
+> **Note**
+>
+> The name of the secret and the data key of the kubeconfig field must match the following example:
+>
+> ```yaml
+> apiVersion: v1
+> kind: Secret
+> metadata:
+>   name: br-federation-kubeconfig
+> type: Opaque
+> data:
+>   kubeconfig: ${encoded-kubeconfig}
+> ```
+
+### Step 2.3: Install BR Federation Manager
+
+This section describes how to install the BR Federation Manager using [Helm 3](https://helm.sh/docs/intro/install/).
+
+- If you prefer to use the default configuration, follow the **Quick deployment** steps.
+- If you want to use a custom configuration, follow the **Custom deployment** steps.
```

Comment:
Generally, please use consistent words for similar cases in documentation, while in blog posts, you can use different words to convey one meaning.
```suggestion
- If you prefer to use a custom configuration, follow the **Custom deployment** steps.
```